tests/Avalonia.Markup.Xaml.UnitTests/Xaml/StyleTests.cs

```diff
@@ -278,5 +278,67 @@ public void Style_Can_Use_Not_Selector()
                 Assert.Equal(Colors.Red, ((ISolidColorBrush)notFoo.Background).Color);
             }
         }
+
+        [Fact]
+        public void Style_Can_Use_Or_Selector_1()
+        {
+            using (UnitTestApplication.Start(TestServices.StyledWindow))
+            {
+                var xaml = @"
+<Window xmlns='https://github.com/avaloniaui'
+             xmlns:x='http://schemas.microsoft.com/winfx/2006/xaml'>
+    <Window.Styles>
+        <Style Selector='Border.foo, Border.bar'>
+            <Setter Property='Background' Value='Red'/>
+        </Style>
+    </Window.Styles>
+    <StackPanel>
+        <Border Name='foo' Classes='foo'/>
+        <Border Name='bar' Classes='bar'/>
+        <Border Name='baz' Classes='baz'/>
+    </StackPanel>
+</Window>";
+                var loader = new AvaloniaXamlLoader();
+                var window = (Window)loader.Load(xaml);
+                var foo = window.FindControl<Border>("foo");
+                var bar = window.FindControl<Border>("bar");
+                var baz = window.FindControl<Border>("baz");
+
+                Assert.Equal(Brushes.Red, foo.Background);
+                Assert.Equal(Brushes.Red, bar.Background);
+                Assert.Null(baz.Background);
+            }
+        }
+
+        [Fact]
+        public void Style_Can_Use_Or_Selector_2()
+        {
+            using (UnitTestApplication.Start(TestServices.StyledWindow))
+            {
+                var xaml = @"
+<Window xmlns='https://github.com/avaloniaui'
+             xmlns:x='http://schemas.microsoft.com/winfx/2006/xaml'>
+    <Window.Styles>
+        <Style Selector='Border,Canvas,ListBox'>
```

Comment:
wow.. i didnt know this was a thing already! this removes so much redundant styles in many apps out there.

Reply:
<img src="https://user-images.githubusercontent.com/17993847/77015121-82660d80-6939-11ea-8d80-a2c1975d7424.jpg" alt="drawing" width="300"/>